### PR TITLE
Remove useUdp from the code and code used only when useUDP is passed

### PR DESCRIPTION
**Pull request by jakubhava**
(cherry picked from commit d516589ecef868eeb8ebbed0cca94e246d6667bc)

### CODE REVIEW

**Comment by jakubhava** [comment]
@michalkurka super core change, but initial attempt to remove UDP from codebase. Understand if we don't merge it, but here's the PR 👍

File: h2o-core/src/main/java/water/H2O.java [hunk 1]
```diff
@@ -564,9 +562,6 @@ else if (s.matches("md5skip")) {
       else if (s.matches("quiet")) {
         trgt.quiet = true;
       }
-      else if(s.matches("useUDP")) {
-        trgt.useUDP = true;
```
Comment (michalkurka) [inline h2o-core/src/main/java/water/H2O.java]: Can we provide an explanatory message here?

**Comment by michalkurka** [comment]
@jakubhava looks good to me, I would just add an error message if someone used the deprecated parameter, or a warning

I am also adding @tomkraljevic and @mmalohlava to the review. I think removing true UDP is fine now but would be good to get more opinions.

File: h2o-core/src/main/java/water/H2O.java (hunk 1 above)
Comment (jakubhava) [reply h2o-core/src/main/java/water/H2O.java]: @michalkurka sure, thanks!

**Comment by michalkurka** [comment]
@tomkraljevic / @mmalohlava - I don't think you need to go over the code and the changes. I just want to confirm you don't know of any use case for `useUdp`.

**Pull request merged**